101-recovery-services-vm-workload-backup/azuredeploy.json

```diff
@@ -0,0 +1,193 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vaultName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Vault"
+      }
+    },
+    "vmResourceGroup": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().name]",
+      "metadata": {
+        "description": "Resource group of Compute VM containing the workload"
+      }
+    },
+    "vmName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Compute VM containing the workload"
+      }
+    },
+    "policyName": {
+      "type": "string",
+      "metadata": {
+        "description": "Backup Policy Name"
+      }
+    },
+    "databaseInstanceName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of database server instance"
+      }
+    },
+    "databaseName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of protectable data source i.e. Database Name"
+      }
+    },
+    "isNewVault": {
+      "type": "bool",
+      "defaultValue": false,
+      "metadata": {
+        "description": "Conditional parameter for New or Existing Vault"
+      }
+    },
+    "isNewPolicy": {
+      "type": "bool",
+      "defaultValue": false,
+      "metadata": {
+        "description": "Conditional parameter for New or Existing Backup Policy"
+      }
+    },
+    "workloadType": {
+      "type": "string",
+      "defaultValue": "SQLDataBase",
+      "allowedValues": [
```

Comment:
if there is only one allowedValue, why is this a parameter?

Reply:
planning to add support for more workloads as allowed values, like SAP HANA and ASE